content/minfraud/api-documentation/responses.mdx

```diff
@@ -0,0 +1,229 @@
+### Error Body
+<Example language="json">
+  {JSON.stringify(responseJson.error, null, 2)}
+</Example>
+
+## Errors
+
+When the server returns an error (`4xx` or `5xx`), the response may include a
+JSON document in the body. This document is a single object with the keys `code`
+and `error`. The `code` field is a static error code for machine use. The value
+of any given code will never change, though codes can be added or removed. The
+`error` field is a human-readable description of the error and may change at any
+time.
+
+Not all errors include a JSON body. An error in content negotiation will not
+include a body, nor will many `5xx` errors, which typically happen outside of
+our web service request handling code. You should check the `Content-Type` type
+of an error response before attempting to decode the body as JSON.
+
+In addition to the errors documented below, client code should also be prepared
+to handle any valid HTTP `4xx` or `5xx` status code.
+
+<table>
+  <thead>
+    <tr>
+      <th>Code</th>
+      <th>HTTP Status</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>`JSON_INVALID`</td>
+      <td>400 Bad Request</td>
+      <td>We cannot decode the body as a JSON object.</td>
+    </tr>
+    <tr>
+      <td>`REQUEST_INVALID`</td>
+      <td>400 Bad Request</td>
+      <td>
+        The request body is valid JSON but contains no valid input values.
+      </td>
+    </tr>
+    <tr>
+      <td>`AUTHORIZATION_INVALID`</td>
+      <td>401 Unauthorized</td>
+      <td>
+        You have supplied an invalid [MaxMind account ID and/or license key](https://www.maxmind.com/en/my_license_key)
+        in the [Authorization](/minfraud/api-documentation/requests#authorization-and-security)
+        header.
+      </td>
+    </tr>
+    <tr>
+      <td>`LICENSE_KEY_REQUIRED`</td>
+      <td>401 Unauthorized</td>
+      <td>
+        You have not supplied a [MaxMind license key](https://www.maxmind.com/en/my_license_key)
+        in the [Authorization](/minfraud/api-documentation/requests#authorization-and-security)
+        header.
+      </td>
+    </tr>
+    <tr>
+      <td>`ACCOUNT_ID_REQUIRED`</td>
+      <td>401 Unauthorized</td>
+      <td>
+        You have not supplied a [MaxMind account ID](https://www.maxmind.com/en/my_license_key)
+        in the [Authorization](/minfraud/api-documentation/requests#authorization-and-security)
+        header.
+      </td>
+    </tr>
+    <tr>
+      <td>`INSUFFICIENT_FUNDS`</td>
+      <td>402 Payment Required</td>
+      <td>
+        The license key you have provided does not have sufficient funds to use
+        this service. Please [purchase more service credits](https://www.maxmind.com/en/purchase-minfraud-services).
+      </td>
+    </tr>
+    <tr>
+      <td>`PERMISSION_REQUIRED`</td>
+      <td>402 Payment Required</td>
+      <td>
+        You do not have permission to use the service. Please contact
+        support@maxmind.com for more information.
```

Comment:
`support@maxmind.com` displays with a dotted line underneath, but the link doesn't bring up the email client